<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-06-05T11:20:21 IST" name="Default suite" finished-at="2023-06-05T11:20:22 IST" duration-ms="759">
    <groups>
    </groups>
    <test started-at="2023-06-05T11:20:21 IST" name="Default test" finished-at="2023-06-05T11:20:22 IST" duration-ms="759">
      <class name="driverFactory.AppTest">
        <test-method signature="kickstart()[pri:0, instance:driverFactory.AppTest@795509d9]" started-at="2023-06-05T11:20:21 IST" name="kickstart" finished-at="2023-06-05T11:20:22 IST" duration-ms="709" status="FAIL">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[E:\QEDGE\Automation Testing\Project_Rangareddy\StockAccounting_ERP_1\FileOutput\HybridResults.xlsx (The process cannot access the file because it is being used by another process)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: E:\QEDGE\Automation Testing\Project_Rangareddy\StockAccounting_ERP_1\FileOutput\HybridResults.xlsx (The process cannot access the file because it is being used by another process)
at java.base/java.io.FileOutputStream.open0(Native Method)
at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
at utilities.ExcelFileUtil.setCellData(ExcelFileUtil.java:86)
at driverFactory.DriverScript.startTest(DriverScript.java:68)
at driverFactory.AppTest.kickstart(AppTest.java:10)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- kickstart -->
      </class> <!-- driverFactory.AppTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
